src/client/theme-default/components/VPLocalSearchBox.vue

```diff
@@ -299,26 +299,40 @@ function scrollToSelectedResult() {
   })
 }
 
-onKeyStroke('ArrowUp', (event) => {
-  event.preventDefault()
-  selectedIndex.value--
-  if (selectedIndex.value < 0) {
-    selectedIndex.value = results.value.length - 1
-  }
+function goRelativeItem(offset: number) {
+  const resNum = results.value.length
+  selectedIndex.value = (selectedIndex.value + resNum + (offset % resNum)) % resNum
   disableMouseOver.value = true
   scrollToSelectedResult()
+}
+
+onKeyStroke('ArrowUp', (event) => {
+  event.preventDefault()
+  goRelativeItem(-1)
 })
 
 onKeyStroke('ArrowDown', (event) => {
   event.preventDefault()
-  selectedIndex.value++
-  if (selectedIndex.value >= results.value.length) {
-    selectedIndex.value = 0
-  }
-  disableMouseOver.value = true
-  scrollToSelectedResult()
+  goRelativeItem(1)
 })
 
+const isMac = navigator.platform.toUpperCase().indexOf('MAC') >= 0
```

Comment:
`navigator.platform` has been deprecated, but I want to get whether the modifier key for keyboard shortcuts is the `⌘` command key rather than the `⌃` control key